Makefile

```diff
@@ -16,7 +16,7 @@ all: build-full test lint
 
 test: test-go test-tiny test-cli check-coverage
 
-lint: lint-go lint-sh check-prettier check-style check-fmt-evy conform
+lint: lint-go lint-sh check-style check-fmt-evy conform
```

Comment:
Why has `check-prettier` been removed from this? Was that from experimenting with parallelism and not put back?

Reply:
thanks. fixing.

Reply:
addressed.